sui-execution/v1/sui-adapter/src/programmable_transactions/execution.rs

```diff
@@ -75,6 +77,9 @@ mod checked {
         gas_charger: &mut GasCharger,
         pt: ProgrammableTransaction,
     ) -> Result<Mode::ExecutionResults, ExecutionError> {
+        #[skip_checked_arithmetic]
```

Comment:
A comment here and above explaining that this is pretty printed while the original (without trace!) is kept in a short version might be helpful for long-term comprehension.